tasks/simple-batch-job/README.adoc

```diff
@@ -0,0 +1,221 @@
+:sectnums:
+= Batch Job on Cloud Foundry
+
+In this demonstration, you will learn how to orchestrate short-lived data processing (_eg: Spring Batch Jobs_) using http://cloud.spring.io/spring-cloud-task/[Spring Cloud Task] and http://cloud.spring.io/spring-cloud-dataflow/[Spring Cloud Data Flow] on Cloud Foundry.
+
+== Using Cloud Foundry Server
+
+=== Prerequisites
+
+In order to get started, make sure that you have the following components:
+
+* Local https://pivotal.io/pcf-dev[PCFDev] instance 
+* Local install of https://github.com/cloudfoundry/v3-cli-plugin[V3-CF-CLI] plugin
+* Local build of https://github.com/spring-cloud/spring-cloud-dataflow[Spring Cloud Data Flow]
+* Local build of Spring Cloud Data Flow's https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry[Cloud Foundry Server]
+* Running instance of `mysql` in Cloud Foundry or from another Cloud provider
```

Comment:
If we keep the `PCFDev` component above then we would need to say "Running instance of `mysql` in PCFDev" .   To be consistent.